### ISSUE TITLE
[BUG] Google id avatar not working

### ISSUE THREAD

**Issue by bhavya-giri** [Dharmik48/notes-react-firebase-v2.0#20]
### Description

When we sign in the Gmail accounts' avatar doesn't show up

### Screenshots

<img width="1800" alt="Screenshot 2022-08-22 at 1 05 37 PM" src="https://user-images.githubusercontent.com/102273412/185865811-ddd013bb-8774-4016-a9fa-fdfe8448f5d3.png">

**Comment by Dharmik48** [comment]
Yeah, that a bug from firebase itself, don't know why but sometimes it just doesn't work
